sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -0,0 +1,332 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright(c) 2021 Mediatek Inc. All rights reserved.
+//
+// Author: YC Hung <yc.hung@mediatek.com>
+//
+
+/*
+ * Hardware interface for audio DSP on mt8195
+ */
+
+#include <linux/delay.h>
+#include <linux/firmware.h>
+#include <linux/io.h>
+#include <linux/of_address.h>
+#include <linux/of_irq.h>
+#include <linux/of_platform.h>
+#include <linux/of_reserved_mem.h>
+#include <linux/module.h>
+
+#include <sound/sof.h>
+#include <sound/sof/xtensa.h>
+#include "mt8195.h"
+#include "../adsp_helper.h"
+#include "../mediatek-ops.h"
+#include "../../ops.h"
+#include "../../sof-audio.h"
```

Comment:
I think it is more logical to start from the include which is _further_ from the current dir.
If there are no other change request, no need to change just this.

Reply:
ok. I will reorder them. thanks